src/archiefvernietigingscomponent/destruction/models.py

```diff
@@ -102,29 +103,33 @@ def next_assignee(self, review=None):
         #  all reviews have approve status -> list is about to be completed
         return None
 
-    def assign(self, assignee):
-        self.assignee = assignee
-        is_reviewer = assignee != self.author
+    def assign(self, assigned_user: User) -> None:
 
-        if assignee:
+        self.assignee = assigned_user
+        is_reviewer = assigned_user != self.author
+        assignees = self.assignees.get(assignee=assigned_user)
```

Comment:
```suggestion
        assignee = self.assignees.get(assignee=assigned_user)
```
Since it will be only one assignee